samples/dashboards/demo/vertical/demo.js

```diff
@@ -0,0 +1,272 @@
+const weatherEventsMap = [
+    'Rare',
+    'Uncommon',
+    'Increasing',
+    'More frequent',
+    'Frequent',
+    'Much more frequent',
+    'Common',
+    'Common, intense',
+    'Extreme'
+];
+
+Highcharts.setOptions({
+    chart: {
+        animation: false
+    },
+    credits: {
+        enabled: false
+    },
+    title: {
+        text: ''
+    },
+    tooltip: {
+        shared: true
+    }
+});
+
+const pointFormatter = function () {
+    const series = this.series;
+    return `<span style="color:{point.color}">\u25CF</span>
+    ${series.name}:<b>${weatherEventsMap[this.y]}</b><br/>`;
+};
+
+Dashboards.board('container', {
+    dataPool: {
+        connectors: [{
+            id: 'climate-data',
+            type: 'JSON',
+            options: {
+                firstRowAsNames: false,
+                orientation: 'columns',
+                columnNames: [
+                    'Decade',
+                    'Global Avg Temperature Increase',
+                    'CO2 Concentration',
+                    'Sea Level Rise',
+                    'Extreme Weather Events',
+                    'Mapped events'
+                ],
+                data: [
+                    [
+                        '1920-1929',
+                        '1930-1939',
+                        '1940-1949',
+                        '1950-1959',
+                        '1960-1969',
+                        '1970-1979',
+                        '1980-1989',
+                        '1990-1999',
+                        '2000-2009',
+                        '2010-2019'
+                    ],
+                    [0.0, 0.1, 0.1, 0.2, 0.2, 0.3, 0.4, 0.5, 0.6, 0.8],
+                    [305, 310, 312, 315, 320, 327, 340, 355, 375, 400],
+                    [0.0, 0.2, 0.4, 0.8, 1.0, 2.0, 3.0, 4.5, 6.0, 8.0],
+                    [
+                        'Rare',
+                        'Uncommon',
+                        'Uncommon',
+                        'Increasing',
+                        'More frequent',
+                        'Frequent',
+                        'Much more frequent',
+                        'Common',
+                        'Common, intense',
+                        'Extreme'
+                    ]
+                ],
+                beforeParse: function (data) {
+                    data.push(data[4].map(function (event) {
+                        return weatherEventsMap.indexOf(event);
```

Comment:
Hm..... I don't like this weatherEventsMap, since it's 1:1 identical copy of stuff we already got in the dataset.  Possible to use the values from dataset instead, or some other mechanism to achieve the same?

Reply:
It's not an identical copy, but if we have data in this format we have to map it somehow. 
To show values on the chart we can't use strings that is why we have to create a column here with mapped strings to numbers.

Reply:
Can you use` _.uniq` (or `filter`) to remove that duplicate Uncommon from the original instead of hardcoding a copy?